ddp-workspace/projects/ddp-sdk/src/lib/services/sessionStorage.service.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { TestBed } from '@angular/core/testing';
+import { SessionStorageService } from './sessionStorage.service';
+
+describe('SessionStorageService', () => {
+    let service: SessionStorageService;
+    let sessionStorageMock: Storage;
+
+    const sessionItem = 'sessionResult';
+    beforeEach(() => {
+        sessionStorageMock = {
+            getItem: jasmine.createSpy('getItem').and.returnValue(sessionItem),
+            setItem: jasmine.createSpy('setItem'),
+            removeItem: jasmine.createSpy('removeItem'),
+            clear: jasmine.createSpy('clear'),
+        } as unknown as Storage;
```

Comment:
` as unknown as Storage`
what can it mean?

Reply:
when described object differs from the interface simple assertion doesn't help, "as unknown as" is a known trick in unit tests to provide some mock data with only necessary properties for complicated interfaces; it's also might be used in the code when there are difficulties with types compatibility from 3d party libraries.

Reply:
Cool!
I did not know that.